packaging/ios/build.sh

```diff
@@ -44,10 +44,10 @@ fi
 
 # Which versions are we building against?
 SQLITE_VERSION=3090200
-LIBXML2_VERSION=2.9.2
-LIBXSLT_VERSION=1.1.28
-LIBZIP_VERSION=0.11.2
-LIBGIT2_VERSION=0.26.0
+#LIBXML2_VERSION=2.9.2
+LIBXSLT_VERSION=1.1.32
+LIBZIP_VERSION=1.2.0
+LIBGIT2_VERSION=v0.27.0
 
```

Comment:
I totally agree to that, but I had to do it in 2 PR, because this one is about a change in ios/build.sh and just got enhanced with the libgit2 change.

The other PR is only the libgit2 change in scripts/build.sh

So when both PR's are merged we use libgit2 overall.

Sorry for making 2 PR, but I felt it would be wrong to add scripts/build.sh in this PR.

Reply:
i think that the build script and documentation changes should be separate commits unless they intersect - e.g. bound versioning.

i guess my other point is that when library versions are bumped for one OS they should be bumped everywhere - i.e. Windows, Linux and Android too.

Reply:
I agree with that in most cases.
Clearly I'm at fault here because I drove @janiversen in that direction with an earlier comment.